### PR TITLE
[IMP] knowledge : Adapt for dark mode

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This PR fixes and adapts some issues in Knowledge for dark mode: 

- Fix tooltip colors
- Fix Powerbox scrollbar colors
- Adapt colors for `select2` dropdown.

Requires:
- https://github.com/odoo-dev/enterprise/pull/392

task-2710677

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @stefanorigano 👋 ,

I'm working on color adaptations in Knowledge. I'm facing a particular case with the `select2` dropdown list.
As changing the value directly in the library is not a good idea and there is no common scss file to manage the values between light and dark mode, I made a test to fix this:

1. I created a common scss file in `web` (`web/static/src/scss/select2_overriden.scss`). The goal is to overwrite all the color values in this file to handle them in dark mode.
2. I added it in the same bundle as `bootstrap_overriden.scss` in the manifest (`_assets_backend_helpers`).
3. Like most of the dark mode adaptations,  I added another `select2_overriden.scss` in `color_scheme` to change the values for dark mode.

I would like to have your opinion about this, do you think this is a good idea? I am waiting for your feedback to continue (or not) to replace the color values in `web/static/src/scss/select2_overriden.scss`.

File: addons/web/__manifest__.py [hunk 1]
```diff
@@ -564,6 +564,7 @@
         'web._assets_backend_helpers': [
```
Comment (stefanorigano) [inline addons/web/__manifest__.py]: Can't it be placed in `_assets_backend` ?
This bundle is used for other purposes

Comment (Brieuc-brd) [reply addons/web/__manifest__.py]: @stefanorigano I moved it there : https://github.com/odoo-dev/odoo/pull/1365/files#diff-66d417d1828879bbc2124ddc118b47030ea7c30fb93e660c5231365420002214R194

Is it ok for you ?

File: addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss [hunk 2]
```diff
@@ -300,10 +300,10 @@
         height: 10px;
     }
     ::-webkit-scrollbar-thumb {
-        background: #D3D1CB;
+        background: var(--oePowerbox__ScrollbarThumb-background-color, #D3D1CB);
```
Comment (stefanorigano) [inline addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss:303]: Hello @dmo-odoo , we need these lines to correctly handle dark-mode for the editor :)
If we merge them could you eventually forward the change to the lib?

Note that these are CSS variables so if the user don't define them they will juts fallback to the previous values ;)

File: addons/web/__manifest__.py [hunk 3]
```diff
@@ -191,6 +191,7 @@
             'web/static/src/webclient/icons.scss', # variables required in list_controller.scss
             'web/static/src/views/**/*',
             'web/static/src/webclient/**/*',
+            'web/static/src/scss/select2_overriden.scss',
```
Comment (stefanorigano) [inline addons/web/__manifest__.py]: Yes, that's the bundle.... but I'm now thinking that adding the file here will rise a lot of red flags across the entire department ^^
Sorry for the headache , could you just add the code in webclient.scss ? In retrospective it's the best option

**Review by stefanorigano — CHANGES_REQUESTED** [review]

File: addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss (hunk 2 above)
Comment (dmo-odoo) [reply addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss:303]: Hello @stefanorigano, the editor is not a separate lib anymore so this is fine :-)

Comment (stefanorigano) [reply addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss:303]: thanks @dmo-odoo 👍

**Comment by Brieuc-brd** [comment]
@stefanorigano Ready for review 👍

**Comment by stefanorigano** [comment]
cherry-picked, thanks 👍

**Pull request closed without merging**